deeptime/markov/tools/estimation/api.py

```diff
@@ -213,7 +214,83 @@ def effective_count_matrix(dtrajs, lag, average='row', mact=1.0, n_jobs=None, ca
     return sparse.effective_counts.effective_count_matrix(dtrajs, lag, average=average, mact=mact, n_jobs=n_jobs,
                                                           callback=callback)
 
+def girsanov_reweighted_count_matrix(dtraj, lag, reweighting_factors,
+                                     sliding=True, sparse_return=True, nstates=None):
+    r"""Generate a Girsanov reweighted count matrix from given microstate trajectory. :footcite:`donati2017girsanov`
```

Comment:
This will need an entry in [references.bib](docs/source/references.bib) to render correctly

Reply:
we will add the reference as soon as it is published